htdocs/public/stripe/ipn.php

```diff
@@ -21,7 +21,9 @@
 $entity=(! empty($_GET['entity']) ? (int) $_GET['entity'] : (! empty($_POST['entity']) ? (int) $_POST['entity'] : 1));
 if (is_numeric($entity)) define("DOLENTITY", $entity);
 
-require '../../main.inc.php';
+$res=0;
+if (! $res && file_exists("../../main.inc.php")) $res=@include("../../main.inc.php");		// to work if your module directory is into a subdir of root htdocs directory
```

Comment:
Don't understand this.
ipn.php file is always at same place and the main.inc.php always at same place too compared to ipn.php
So this should be useless.